CHANGELOG.md

```diff
@@ -1,3 +1,73 @@
+## Fleet 4.61.0 (Dec 17, 2024)
+
+### Bug fixes
```

Comment:
```suggestion
```

I think this is a stray "Bug fixes" header. We have the "Bug fixes and improvements" section here: https://github.com/fleetdm/fleet/pull/24407/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR25